### ISSUE TITLE
apm (Atom) Completions

### ISSUE THREAD

**Issue by dan-c-underwood** [fish-shell/fish-shell#2354]
Current completions for the command `apm` seem to refer to a Application Monitoring tool (and were committed in 2007 - [file in question](https://github.com/fish-shell/fish-shell/blob/master/share/completions/apm.fish). Would it be worth changing the completions to refer to the `apm` tool included with the [Atom](https://atom.io) text editor instead?

(The command in question seems to refer to the [apmd](https://packages.debian.org/wheezy/apmd) package on Linux and some BSD distros)

If so I'm happy to work through and switch it across and submit a PR!

**Comment by faho** [comment]
It seems like it's not possible to install both at the same time.

I'd merge such a completion, but only if it works for any one of the two installed as "apm".

Something like

```fish
if apm --version | grep -q "Atom"
    complete -c apm -s $SOMEOPTIONATOMHAS
else
   complete -c apm -s $SOMEOPTIONAPMDHAS
end
```

should do the trick.

See the "sysctl" completions for a similar trick.

**Comment by dan-c-underwood** [comment]
Thanks! I'll try that and get working on it!

**Comment by faho** [comment]
@dan-c-underwood: Any progress?

**Comment by dan-c-underwood** [comment]
I've been in the middle of moving house over the last week but I'm planning on getting it done today/tomorrow!

**Comment by dan-c-underwood** [comment]
This is now fixed by #2390.
